css/_side_toolbar_container.scss

```diff
@@ -72,16 +74,16 @@
         /**
          * Titles and subtitles of inner containers.
          */
-        > div.title,
-          div.subTitle {
+         div.title,
+         div.subTitle {
             text-align: left;
-            margin: 10px 0px 10px 0px;
+            margin: 10px 0;
         }
 
         /**
          * Main title size.
          */
-        > div.title {
+        div.title {
             color: $defaultColor !important;
             text-align: center;
```

Comment:
Yep. It overwrites stiles above. I don't think it discord. I can rewrite it separately and add one more rule for `div.subTitle `